dirty_cat/VERSION.txt

```diff
@@ -1 +1 @@
-0.4.dev0
+0.4.0
```

Comment:
Hum, that changes seems wrong: it should be 0.4.1.dev0, I believe

Reply:
Yes will change, it was just to test if this impacts the rendering!